libraries/api-extractor/package.json

```diff
@@ -28,7 +28,6 @@
   },
   "dependencies": {
     "@microsoft/node-core-library": "~0.2.5",
-    "@types/es6-collections": "0.5.29",
```

Comment:
After this change, we should merge this definition:

[https://github.com/Microsoft/web-build-tools/tree/master/core-build/gulp-core-build-typescript/resources/external-api-types/es6-collections](https://github.com/Microsoft/web-build-tools/tree/master/core-build/gulp-core-build-typescript/resources/external-api-types/es6-collections)

into this file:

[https://github.com/Microsoft/web-build-tools/tree/master/core-build/gulp-core-build-typescript/resources/external-api-types/web-apis](https://github.com/Microsoft/web-build-tools/tree/master/core-build/gulp-core-build-typescript/resources/external-api-types/web-apis)

Could you open a PBI for that and assign it to me?